File: Discord_win/Discord_win/Models/ServerLevelPermission.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Discord_win.Models {
    public class ServerLevelPermission {
        public int RoleId { get; set; }
        public int PermissionId { get; set; }
        public bool IsActive { get; set; }

        public Role Role { get; set; }
        public ServerPermission ServerPermission { get; set; }
    }
}

File: Discord_android/Discord/app/src/main/java/com/peanut/discord/models/ServerLevelPermission.java
package com.peanut.discord.models;
public class ServerLevelPermission {
    private int roleId;
    private int permissionId;
    private boolean isActive;
    private Role role;
    private ServerPermission serverPermission;
    public int getRoleId() {
        return roleId;
    }
    public void setRoleId(int roleId) {
        this.roleId = roleId;
    }
    public int getPermissionId() {
        return permissionId;
    }
    public void setPermissionId(int permissionId) {
        this.permissionId = permissionId;
    }
    public boolean isActive() {
        return isActive;
    }
    public void setActive(boolean active) {
        isActive = active;
    }
    public Role getRole() {
        return role;
    }
    public void setRole(Role role) {
        this.role = role;
    }
    public ServerPermission getServerPermission() {
        return serverPermission;
    }
    public void setServerPermission(ServerPermission serverPermission) {
        this.serverPermission = serverPermission;
    }
}
